### ISSUE TITLE
Typecheck failing – generated TS definition file appears to be invalid?

### ISSUE THREAD

**Issue by sebpowell** [Mokshit06/typewind#56]
Hi there, 

Thanks for this package, I was just playing around with it and it looks really great. 

I've been having an issue however when running a type check on my project. I was able to get everything working, but when I run a typecheck like so: `tsc --noEmit`, I get the following error:

<img width="1090" alt="Screenshot 2023-12-30 at 14 53 49" src="https://github.com/Mokshit06/typewind/assets/1786366/d0f00593-aa0c-4393-b148-79a0e8967955">

If I look at the file in questions, it doesn't seem to be valid TS:

<img width="457" alt="Screenshot 2023-12-30 at 14 54 35" src="https://github.com/Mokshit06/typewind/assets/1786366/83229be8-fd71-4b0a-baf3-6aa61891bdc8">

I initially thought it might be an issue with my project, but I was able to replicate this error using the the Next.js example provided here: https://typewind.vercel.app/docs/examples/next

I downloaded the repo, and then tried running the typecheck as described above, and got the same error. 

Any ideas what the problem might be? As far as I'm aware, the syntax the package is generating is not valid TS, irrespective of version, so not sure what the problem is. Thank you in advance.

**Comment by I7ace** [comment]
Downgrade the tailwind version to 3.3.*

**Comment by mutewinter** [comment]
#57 up with a fix for Tailwind 3.4. You can use my PR right now if needed:

```sh
# Yarn 2+
yarn add 'typewind@https://gitpkg.now.sh/mutewinter/typewind/packages/typewind?fix-tailwind-3.4-build'

# Yarn 1
yarn add 'https://gitpkg.now.sh/mutewinter/typewind/packages/typewind?fix-tailwind-3.4-build'

# NPM
npm install 'https://gitpkg.now.sh/mutewinter/typewind/packages/typewind?fix-tailwind-3.4-build'
```

**Comment by Mokshit06** [comment]
Hi, thanks for reporting! This is definitely an issue with Typewind, and #57 fixes it 🙂
